.github/workflows/benchmark.yml

```
@@ -0,0 +1,36 @@
+  workflow_dispatch: # Allow manual execution
+  push:
+  pull_request:
+
+jobs:
+  benchmark:
+    name: Run SQL Protocol Benchmark
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)